app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if not parts or parts[0] != "mv" or len(parts) != 3:
+        return
+    source_file = parts[1]
+    destination = parts[2]
```

Comment:
You can use unpacking
```suggestion
    _, source_file, destination = parts
```